tests/test_threadpoolctl.py

```diff
@@ -388,3 +390,45 @@ def test_libomp_libiomp_warning(recwarn):
     assert "Found Intel" in str(wm.message)
     assert "LLVM" in str(wm.message)
     assert "multiple_openmp.md" in str(wm.message)
+
+
+def test_command_line_empty():
+    output = subprocess.check_output(
+        "python -m threadpoolctl".split())
+    assert json.loads(output.decode("utf-8")) == []
+
+
+def test_command_line_command_flag():
+    pytest.importorskip("numpy")
+    output = subprocess.check_output(
+        ["python", "-m", "threadpoolctl", "-c", "import numpy"])
+    cli_info = json.loads(output.decode("utf-8"))
+
+    this_process_info = threadpool_info()
+    for module in cli_info:
+        assert module in this_process_info
+
+
+@pytest.mark.skipif(sys.version_info < (3, 7),
+                    reason="need recent subprocess.run options")
+def test_command_line_import_flag():
+    result = subprocess.run([
+        "python", "-m", "threadpoolctl", "-i",
+        "numpy",
+        "scipy.linalg",
+        "invalid_package",
+        "numpy.invalid_sumodule",
+    ], capture_output=True, check=True, encoding="utf-8")
+    cli_info = json.loads(result.stdout)
+
+    this_process_info = threadpool_info()
+    for module in cli_info:
+        assert module in this_process_info
```

Comment:
I am unsure how the output will be the same here due to `scipy.linalg`. Is it imported in the `pytest` process? If not, it might have different modules no? Maybe it would be better to import it here to make sure we get the same result?

Reply:
I don't understand: how could the scipy imported by a subprocess call be different from the scipy imported from the parent process?

Reply:
Sorry my question was unclear. My question was: is `scipy.linalg` imported in the parent process. But upon further reading, it is at the beginning of this file if it is present. So nevermind!

Reply:
And the tests pass :)